### DISCUSSION TITLE
Top of Descent in ND

### DISCUSSION THREAD

**Discussion by Vulkan80** [flybywiresim/aircraft#3484] [Help] [answered]
How do I get my TOD in the ND? I setup the MCDU via Simbrief and everything looks right, but I don't get a TOD or TOC. What am I missing? DO I need to takeoff before that shows? Shouldn't it be seen in the plan view right away?

Also is the A320 equiped with the same typ of "arrow" in the ND like the 737 has when you are climbing or descending, so you can see where on your path you will meet your new altitude?

Thanks!

**Comment by ChruutvoLuzi** [comment, marked as answer]
VNAV is not implemented yet

**Comment by robinottawa** [comment]
I get an arrow/hockey stick for my TOC but not for my TOD (custom version).
